plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -62,28 +69,30 @@ open class GenerateGramineBundle @Inject constructor(
     val outputDir: DirectoryProperty = objects.directoryProperty()
 
     override fun action() {
-        enclaveJar.copyToOutputDir(GRAMINE_ENCLAVE_JAR)
+
         if (pythonFile.isPresent) {
             pythonFile.copyToOutputDir(PYTHON_FILE)
         }
 
         // TODO We're relying on gcc, python3, pip3 and jep being installed on the machine that builds the Python
         //  enclave. https://r3-cev.atlassian.net/browse/CON-1181
 
-        val architecture = commandWithOutput("gcc", "-dumpmachine")
+        val architecture = retrieveArchitecture()
         val ldPreload = executePython(
             "from sysconfig import get_config_var; " +
                     "print(get_config_var('LIBPL') + '/' + get_config_var('LDLIBRARY'))"
         )
         // The location displayed by 'pip3 show jep' is actually of the site/dist-packages dir, not the specific 'jep'
         // dir within it. We assume this is the packages dir for other modules as well. If this assumption is
         // incorrect then we'll need to come up with a better solution.
-        val pythonPackagesPath = commandWithOutput("pip3", "show", "jep")
+        val pythonPackagesPath = linuxExec.execWithOutput(listOf("pip3", "show", "jep"))
```

Comment:
Leave python as is. we don't want it executing inside the container in this PR

Reply:
Ok, but what we do with it then? I.e. should we remove Python related code? Should we leave the current flow without running things inside Docker?

Reply:
Yes, leave things as they are for Python and not have it execute inside the container. It's captured in a separate [ticket](https://r3-cev.atlassian.net/browse/CON-1229), which may or may not be done as part of 1.4.

Reply:
ok